stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -428,6 +428,26 @@ extension AsyncStream.Continuation {
   }
 }
 
+@available(SwiftStdlib 5.1, *)
+extension AsyncStream {
+  /// Initializes a new ``AsyncStream`` and an ``AsyncStream/Continuation``.
+  ///
+  /// - Parameters:
+  ///   - elementType: The element type of the stream.
+  ///   - limit: The buffering policy that the stream should use.
+  /// - Returns: A tuple containing the stream and its continuation. The continuation should be passed to the
+  /// producer while the stream should be passed to the consumer.
+  @backDeployed(before: SwiftStdlib 5.9)
```

Comment:
This is missing

```suggestion
  @available(SwiftStdlib 5.1, *)
  @backDeployed(before: SwiftStdlib 5.9)
```

Reply:
Since we removed the requirement that `@available` be present simultaneously with `@backDeployed` this is not technically necessary. `makeStream` will inherit its `introduced:` availability from the enclosing extension so the declaration will have appropriate availability. The suggested change here is unnecessary but could be preferred stylistically.

Reply:
If the ABI checker is complaining, it may be being overly aggressive here.

Reply:
Oh yeah I remember now. Yeah this is a limitation of the ABI checker. Annotating the individual APIs with availability as Konrad has suggested doesn't hurt and it does add clarity for readers so it's the right way to go.

Reply:
Sounds good, thanks for conforming!